.github/workflows/python-formatting.yml

```diff
@@ -0,0 +1,28 @@
+name: Enforce Python Code Formatting
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  format-check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
```

Comment:
Given that the docker image uses python 3.12 wouldn't make more sense to use the same version explicitly?

Reply:
@tibor19 I think its good to be consistent with the other checks using 3.12.2, so i have updated this in the new PR https://github.com/WebOfTrust/keria/pull/280